javascript/javascript1/week2/flight.js

```diff
@@ -0,0 +1,28 @@
+//Flight 
+const firstname= "Mercedes "
+const surname= "Ubeira"
+
+function getFullName(firstname, surname) {
+  console.log (firstname + surname);
+}
+
+getFullName (firstname, surname);
+
+let fullname1= "Emiliano Ubeira";
+let fullname2= " Lionel Messi";
+
+getFullName (fullname1 , fullname2);
+
+const userFormalName = true
```

Comment:
I have a feeling you're trying to trick me :P
I think the idea was to have a function and call it

Reply:
I would expect on line 15 : 
function getNewFullName(firstname, surname, gender) {